### DISCUSSION TITLE
Improvements to Clean Device Storage option

### DISCUSSION THREAD

**Discussion by I-I-IT** [ente-io/ente#822] [Enhancements]
**Describe the feature**
A clear description of what the feature is.
There should be options, for example only delete pictures older than 30 days, only delete videos, etc. 

I don't want to delete everything, just the 50% less useful stuff.

Also please improve the video watching. Currently there is a bug while when the video is loaded you can't share it or view file info.

**Comment by edraac** [comment]
I have Media dating back to 2021 on this phone. I have migrated iPhones twice since then, media was transferred between phones directly. I‘m now reaching 80% of storage usage I just went into Ente to free up device storage, only to find out it’s everything or nothing. I was thinking something like „delete everything older than 2 years“ would be nice, even let me pick how far back I want to keep locally.
